packages/frontend/preview/src/lib/lpc/lpc_preview.svelte

```
@@ -0,0 +1,270 @@
+    zoom?: number;
+    initialState?: LpcPreviewState;
+    onStateChange?: (state: LpcPreviewState) => void;
+    controls?: boolean;
+  };
+
+  let {
+    resolver,
+    allSlots,
+    width = 960,
+    height = 540,
+    zoom = 1,
```

Biome (2.5.7)

[error] This variable zoom is unused.

(lint/correctness/noUnusedVariables)